templates/admin/monitor.tmpl

```diff
@@ -42,6 +42,7 @@
 						<th>{{.i18n.Tr "admin.monitor.desc"}}</th>
 						<th>{{.i18n.Tr "admin.monitor.start"}}</th>
 						<th>{{.i18n.Tr "admin.monitor.execute_time"}}</th>
+						<th></th>
```

Comment:
Intentional?

Reply:
The other rows now have an additional cell with the new cancel button - If you don't have enough header cells it ends up looking weird usually.